map_gui/src/load.rs

```diff
@@ -296,13 +341,21 @@ mod wasm_loader {
                 return (self.on_load.take().unwrap())(ctx, app, &mut timer, result);
             }
 
-            self.panel = ctx.make_loading_screen(Text::from_multiline(vec![
+            let mut lines = vec![
                 Line(format!("Loading {}...", self.url)),
                 Line(format!(
                     "Time spent: {}",
                     Duration::realtime_elapsed(self.started)
                 )),
-            ]));
+            ];
+            if let Some(total) = self.total_bytes {
```

Comment:
Many improvements to the loading screen possible -- percentages, using KB or MB units, etc